src/emulator/gui/src/game_selector.cpp

```diff
@@ -44,25 +44,38 @@ void draw_game_selector(HostState &host) {
             ImVec2(0, 0), display_size);
     }
 
+    constexpr float default_icon_size = 64.0f;
+    float icon_size = default_icon_size;
+
+    switch (host.cfg.size_icon){ 
+    case 1: icon_size = icon_size / 2; break;
```

Comment:
Don't use [magic values](https://www.wikiwand.com/en/Magic_number_(programming)), use an `enum` here, for example:
```c
enum IconSize {
    ICON_SIZE_SMALL,
    ICON_SIZE_DEFAULT,
    ICON_SIZE_MEDIUM,
    ICON_SIZE_LARGE,
};
```
put this enum somewhere that you can include both from here and from `config.cpp`
and cast the int to it here in the `switch` like so
```cpp
const auto icon_size = static_cast<IconSize>(host.cfg.icon_size);
switch (icon_size) {
```